frame/staking/src/mock/voter_bags.rs

```diff
@@ -0,0 +1,100 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Autogenerated voter bag thresholds.
+//!
+//! Generated on 2021-07-05T12:08:52.871368217+00:00
+//! for the test runtime.
+
+/// Existential weight for this runtime.
+#[cfg(any(test, feature = "std"))]
+#[allow(unused)]
+pub const EXISTENTIAL_WEIGHT: u64 = 1;
+
+/// Constant ratio between bags for this runtime.
+#[cfg(any(test, feature = "std"))]
+#[allow(unused)]
+pub const CONSTANT_RATIO: f64 = 2.0000000000000000;
+
+/// Upper thresholds delimiting the bag list.
+pub const THRESHOLDS: [u64; 65] = [
```

Comment:
isn't this a bit of an overkill for tests? I'd just manually use `[5, 20, 50, 100]` and assert against those.

Reply:
I didn't see any particular reason not to use a proper geometric set, and it was easy to generate.